src/meta/src/controller/streaming_job.rs

```diff
@@ -1271,9 +1269,7 @@ impl CatalogController {
 
         let txn = inner.db.begin().await?;
 
-        let parallel_unit_to_worker = get_parallel_unit_to_worker_map(&txn).await?;
-
-        let mut fragment_mapping_to_notify = vec![];
+        let fragment_mapping_to_notify = vec![];
```

Comment:
Hmmm, it's always empty and the mapping is not synced to FE, which results in always singleton for batch query.